features/merge_articles.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Merge articles
```

Comment:
These scenarios are nicely set up. What do you think you'd want to add once the merge was fully implemented?